src/main/groovy/com/bmuschko/gradle/docker/tasks/AbstractDockerRemoteApiTask.groovy

```diff
@@ -167,6 +172,17 @@ abstract class AbstractDockerRemoteApiTask extends DefaultTask {
         dockerClient
     }
 
+    private DockerCmdExecFactory getDockerCmdExecFactory() {
+        boolean useNettyExecFactory = Boolean.valueOf(getProject().findProperty('gdpNettyExecFactory').toString())
+            ?: Boolean.valueOf(System.getProperty('gdp.netty.exec.factory'))
+            ?: Boolean.valueOf(System.getenv('GDP_NETTY_EXEC_FACTORY'))
+
+        if (useNettyExecFactory) {
+            this.logger.debug("Using " + NettyDockerCmdExecFactory.class.simpleName + " as driver for " + DockerClient.class.simpleName)
+        }
+        return useNettyExecFactory ? new NettyDockerCmdExecFactory() : null
```

Comment:
FWIW: all timeouts default to infinity if not specified. I probably should not have mentioned that for fear of confusion but just putting it out there for completeness sake should we ever stumble back upon this issue down the road.

Reply:
That's fine for now.